### PR TITLE
epizod: site change

### PR DESCRIPTION
**Pull request by ilike2burnthing**
Technically, I don't think this is so much a site change as just me finding exceptions upon exceptions. There doesn't seem to be a consistent way of identifying categories, it's a mess, and this is my attempt of making at least some sense of it.

One thing I couldn't figure out was how/if I can translate the multiple `category:` selectors to the `cat:` section, to get vague sizes. I tried adding them all, but that didn't work.

### CODE REVIEW

File: src/Jackett.Common/Definitions/epizod.yml [hunk 1]
```diff
@@ -58,7 +58,7 @@ search:
     s: "{{ .Keywords }}"
 
   rows:
-    selector: article:not(:has(a[title="DMCA"]))
+    selector: li[class^="ilovewp-post"]
```
Comment (garfield69) [inline src/Jackett.Common/Definitions/epizod.yml:61]: Eh!
are you using https://wwv.epizod.tv/ or someplace else?
I see 
```html
<article id="post-177920">
	<div class="post-preview">
		<p class="post-meta">
			<span class="posted-on">
				<span class="genericon genericon-time" aria-hidden="true"></span>
				<time class="entry-date published" datetime="2020-12-09T12:52:54+00:00">
					<a href="https://wwv.epizod.tv/macgyver-s05e01-vostfr-hdtv/" title="MacGyver S05E01 VOSTFR HDTV">décembre 9, 2020</a>
				</time>
			</span>
		</p>
		<!-- .post-meta -->
		<h2 class="title-post">
			<a href="https://wwv.epizod.tv/macgyver-s05e01-vostfr-hdtv/" rel="bookmark">MacGyver S05E01 VOSTFR HDTV</a>
		</h2>
		<p class="post-excerpt">Agé d’une vingtaine d’années, MacGyver, qui possède un don unique pour la débrouille et des méthodes peu orthodoxes pour parvenir à ses fins, travaille pour le gouvernement au sein d’une…
			<span class="read-more-span">
				<a href="https://wwv.epizod.tv/macgyver-s05e01-vostfr-hdtv/" class="more-link">Torrent
					<span class="screen-reader-text">
						"MacGyver S05E01 VOSTFR HDTV"</span>
					<span class="genericon genericon-next" aria-hidden="true"></span>
				</a>
			</span>
		</p>
	</div>
	<!-- .post-preview -->
</article>
```

Comment (garfield69) [reply src/Jackett.Common/Definitions/epizod.yml:61]: oh, I see. you went a level out. ok.

**Review by garfield69 — CHANGES_REQUESTED** [review]
if there are so many possible selectors for the category, then something like
```yaml
[...]
  fields:
    category1:
      selector: a[href*="/category/"]
      attribute: href
      optional: true
      filters:
        - name: split
          args: ["/", 4]
    category2:
      selector: li[class^="ilovewp-post"]
      attribute: class
      optional: true
      filters:
        - name: regexp
          args: "category-(.+?) "
    category3:
      selector: img[src*="img"]
      attribute: src
      optional: true
      filters:
        - name: regexp
          args: "img/(.+?)_"
    category:
      text: "{{ if or .Result.category1 .Result.category2 .Result.category3 }}{{ or .Result.category1 .Result.category2 .Result.category3 }}{{ else }}other{{ end }}"
    sizecat:
      text: "{{ if or .Result.category1 .Result.category2 .Result.category3 }}{{ or .Result.category1 .Result.category2 .Result.category3 }}{{ else }}other{{ end }}"
    site_date:
[...]
          args: "2006-01-02 15:04:05-07:00"
    size:
      text: "{{ if eq .Result.sizecat \"ebook\" }}1 MB{{ else }}{{ end }}{{ if eq .Result.sizecat \"musique\" }}128 MB{{ else }}{{ end }}{{ if or (eq .Result.sizecat \"series\") (eq .Result.sizecat \"other\") }}512 MB{{ else }}{{ end }}{{ if eq .Result.sizecat \"films\" }}2 GB{{ else }}{{ end }}"
    seeders:
[...]
```

**Comment by ngosang** [comment]
@ilike2burnthing Could you test to add user-agent header?
`Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/81.0.4044.138 Safari/537.36`
https://github.com/FlareSolverr/FlareSolverr/issues/25#issuecomment-744290187

**Comment by ilike2burnthing** [comment]
@ngosang copied from https://github.com/Jackett/Jackett/blob/167002f183f6df712fb41e2bbeaeb5eacfbdb04c/src/Jackett.Common/Definitions/cpabien.yml

Had to try the inverse of this, as I'm testing in Windows Sandbox, and use `Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/87.0.4280.88 Safari/537.36`.

Without using FlareSolverr, and with or without also adding a cookie, searches in Jackett were successful (I'm assuming this should fail). Changing to the same user-agent in browser results in the `Access Denied` page for https://wwv.epizod.tv/

**Comment by ilike2burnthing** [comment]
Tested on my NAS as well, equivalent results when using `Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/81.0.4044.138 Safari/537.36` - still blocked.

**Review by garfield69 — APPROVED** [review]

**Review by garfield69 — APPROVED** [review]

**Pull request merged**